example-app/app/Models/Airline.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Airline extends Model
+{
+    use HasFactory;
+
+    public function cities()
+    {
+        return $this->belongsToMany(City::class, 'city_airlines')->using(City_Airline::class);
```

Comment:
There is no need to define the pivot class while following naming convention for table names

https://laravel.com/docs/11.x/eloquent-relationships#many-to-many-table-structure

But there is no problem if you want to keep it explicitly